src/app/about-us/mission-vision/mission-vision.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MissionVisionComponent } from './mission-vision.component';
+import { TranslateModule, TranslateLoader, TranslateFakeLoader } from '@ngx-translate/core';
```

Comment:
Imports order goes like this:
Angular imports first (these start with @angular)

External imports second

Own imports last